mainnet/13-wysiwyg-finance-transfer/metadata.json

```diff
@@ -0,0 +1,12 @@
+{
+  "voting": "0x0e835020497b2cd716369f8fc713fb7bd0a22dbf",
+  "submitterJustification": "If approved, this proposal will transfer 1 ETH to the Aragon Association multisig address 0xcafe1a77e84698c83ca8931f54a755176ef75f2c",
+  "challengerJustification": "ipfs:QmNxuAgYZTvRE2W7xEMzE2Lbitfbpax42QYYPANy6WXuqP",
```

Comment:
In terms of this, is this the `against.md` file's CID?

Reply:
Yes, this would be interesting. I can give it a try :)